dcpy/test/lifecycle/ingest/test_configure.py

```diff
@@ -55,7 +70,64 @@ def test_read_template():
     )
 
 
+@mock.patch("requests.get", side_effect=mock_request_get)
+def test_get_version(mock_request_get, create_buckets):
```

Comment:
with the order-of-args problem you showed me, does it help having the value of `side_effect` be the same as the "could be made up" first arg? I don't remember what you had `side_effect` set to when we were looking at this locally